web/src/js/__tests__/components/Header/__snapshots__/FileMenuSpec.js.snap

```diff
@@ -72,17 +72,9 @@ exports[`FileMenu Component should render correctly 1`] = `
         />
          Options
       </a>
-       
-    </li>
-    <li>
```

Comment:
I think this is because we just upgraded the `react-test-renderer`, some rendering algorithm might have been updated, so it makes a diff... This file is generated automatically by `jest`, so I believe that the diff is innocuous.